pages/common/gh-gist.md

```diff
@@ -3,22 +3,22 @@
 > Work with GitHub Gists on the command-line.
 > More information: <https://cli.github.com/manual/gh_gist>.
 
-- Create a new Gist from a space-separated list of files:
+- Create a new Gist from specific files:
```

Comment:
The old description is much more clear.

Reply:
By default all ellipsis placeholders use space separator, it's standardized format and expectation of how to delimit files. I don't know why not to rely on such standardization.